src/assets/scss/base/_global.scss

```diff
@@ -1,3 +1,24 @@
 body {
     font-size: $font-size-default;
-}
\ No newline at end of file
+    margin: 0;
+}
+
+.main {
+    position: relative;
+    z-index: 100;
+    background-image: linear-gradient(180deg, $primary-gradient, $secondary-gradient, $tertiary-gradient);
+    &:before {
+        content: " ";
+        background-size: cover;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background-image: url('../images/stars.png');
+        z-index: -1; 
+    }
+    &--bright {
```

Comment:
this will change the opacity of the whole page and not just the stars. Try adding it to just the before of --bright.